### DISCUSSION TITLE
How and where are the tutorials being stored?

### DISCUSSION THREAD

**Discussion by pajot** [openego/oeplatform#888] [Q&A] [answered]
Hi everyone:

Are the tutorials themselves stored in the PostgreSQL database, or is this simply static content that you are linking to?

Thanks

Stephen

**Comment by wingechr** [comment, marked as answer]
the tutorials (markdown and notebooks) are stored in the django database afaik. Also, there are some legacy tutorials (notebooks) that are on github that are integrated.

**Reply by jh-RLI** [reply]
JupyterNotebook tutorials are not stored in the database. We use a git submodule -> examples (Repo.: OpenEnergyPlatform/tutorial) which can be updated by commands. In the Django app tutorial the JupyterNotebooks are converted to HTML. So notebooks are always static content.  Additionally there is metadata for each notebook tutorial as JSON file. Only the markdown tutorials are stored in the Django database.
